src/xmipp/bindings/matlab/mirt3D_mexinterp.cpp

```diff
@@ -170,19 +169,21 @@ void mexFunction( int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[] )
     
     /* Get the sizes of each input argument */
     
-    Xndim = mxGetNumberOfDimensions(IN_S);
+    auto Xndim = mxGetNumberOfDimensions(IN_S);
     Xdims = mxGetDimensions(IN_S);
     
     
-    ndim = mxGetNumberOfDimensions(IN_Z);
+    auto ndim = mxGetNumberOfDimensions(IN_Z);
     dims = mxGetDimensions(IN_Z);
     newdims = (size_t*) calloc(ndim-1, sizeof(size_t));
        
-    MN=1;
+    int MN=1;
+    int 1;

```

Comment:
this compiles? I thought that variable names cannot start with number

Reply:
Compile but I dont know why. Resolved.